### PR TITLE
ROX-11892: Enable revive var-naming linting

### PR DESCRIPTION
**Pull request by mtesseract**
## Checklist (Definition of Done)

- [ ] Unit and integration tests added
- [ ] Documentation added if necessary
- [ ] CI and all relevant tests are passing

## Test manual

TODO: Add manual testing efforts

```
# To run tests locally run:
make db/teardown db/setup db/migrate
make ocm/setup OCM_OFFLINE_TOKEN=<ocm-offline-token> OCM_ENV=development
make verify lint binary test test/integration
```

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -49,7 +49,7 @@ linters-settings:
         arguments: ["disableStutteringCheck"] #TODO(ROX-11892) enable revive naming checks
       - name: if-return
       - name: increment-decrement
-      # - name: var-naming #TODO(ROX-11892) enable revive naming checks
+      - name: var-naming #TODO(ROX-11892) enable revive naming checks
```
Comment (vladbologa) [inline .golangci.yml:52]: please remove the TODO and add the ticket in the title

File: internal/dinosaur/pkg/api/admin/private/client.go [hunk 2]
```diff
@@ -188,7 +188,7 @@ func (c *APIClient) ChangeBasePath(path string) {
 	c.cfg.BasePath = path
 }
 
-// GetConfig Allow modification of underlying config for alternate implementations and testing
+// Allow modification of underlying config for alternate implementations and testing
```
Comment (mtesseract) [inline internal/dinosaur/pkg/api/admin/private/client.go:191]: note

File: internal/dinosaur/pkg/api/admin/private/configuration.go [hunk 3]
```diff
@@ -21,7 +21,6 @@ import (
 
 type contextKey string
 
-// String ...
```
Comment (mtesseract) [inline internal/dinosaur/pkg/api/admin/private/configuration.go:24]: note

File: internal/dinosaur/pkg/api/private/configuration.go [hunk 4]
```diff
@@ -26,7 +26,6 @@ func (c contextKey) String() string {
 	return "auth " + string(c)
 }
 
-// ContextOAuth2 ...
```
Comment (mtesseract) [inline internal/dinosaur/pkg/api/private/configuration.go:29]: note

File: internal/dinosaur/pkg/api/public/client.go [hunk 5]
```diff
@@ -191,7 +191,7 @@ func (c *APIClient) ChangeBasePath(path string) {
 	c.cfg.BasePath = path
 }
 
-// GetConfig Allow modification of underlying config for alternate implementations and testing
```
Comment (mtesseract) [inline internal/dinosaur/pkg/api/public/client.go:194]: note

File: internal/dinosaur/pkg/api/public/configuration.go [hunk 6]
```diff
@@ -21,12 +21,10 @@ import (
 
 type contextKey string
 
-// String ...
```
Comment (mtesseract) [inline internal/dinosaur/pkg/api/public/configuration.go:24]: note

File: internal/dinosaur/pkg/api/public/configuration.go [hunk 7]
```diff
@@ -21,12 +21,10 @@ import (
...
 func (c contextKey) String() string {
 	return "auth " + string(c)
 }
 
-// ContextOAuth2 ...
```
Comment (mtesseract) [inline internal/dinosaur/pkg/api/public/configuration.go:29]: note

**Review by mtesseract — COMMENTED** [review]
lgtm

File: pkg/client/observatorium/observability_config.go [hunk 8]
```diff
@@ -10,17 +10,17 @@ import (
...
-	RedHatSsoGatewayUrl        string `json:"redhat_sso_gateway_url" yaml:"redhat_sso_gateway_url"`
-	RedHatSsoAuthServerUrl     string `json:"redhat_sso_auth_server_url" yaml:"redhat_sso_auth_server_url"`
+	RedHatSsoGatewayURL        string `json:"redhat_sso_gateway_url" yaml:"redhat_sso_gateway_url"`
+	RedHatSsoAuthServerURL     string `json:"redhat_sso_auth_server_url" yaml:"redhat_sso_auth_server_url"`
 	RedHatSsoRealm             string `json:"redhat_sso_realm" yaml:"redhat_sso_realm"`
 	RedHatSsoTenant            string `json:"redhat_sso_tenant" yaml:"redhat_sso_tenant"`
-	RedHatSsoTokenRefresherUrl string `json:"redhat_sso_token_refresher_url" yaml:"redhat_sso_token_refresher_url"`
-	MetricsClientId            string `json:"redhat_sso_metrics_client_id" yaml:"redhat_sso_metrics_client_id"`
-	MetricsClientIdFile        string `json:"redhat_sso_metrics_client_id_file" yaml:"redhat_sso_metrics_client_id_file"`
+	RedHatSsoTokenRefresherURL string `json:"redhat_sso_token_refresher_url" yaml:"redhat_sso_token_refresher_url"`
```
Comment (vladbologa) [inline pkg/client/observatorium/observability_config.go:17]: should probably be RedHatSSOTokenRefresherURL

**Review by vladbologa — APPROVED** [review]
lgtm!

**Pull request merged**